server/Html.tsx

```diff
@@ -42,6 +44,12 @@ const Html = (props: Props) => {
 		return `${manifest[`${chunkName}.${extension}`]}`;
 	};
 
+	const activeTargetSlug =
+		props.initialData.scopeData.elements?.activeTarget.slug ||
+		(props.initialData.locationData.path.split('/')[1]
+			? props.initialData.locationData.path.substring(1).replace(/\//gi, '-')
+			: props.initialData.scopeData.elements?.activeTarget.subdomain || 'www');
```

Comment:
Depending on how you look at it, this `www` is a bit of a red herring since we'll never load custom CSS on pubpub.org.